heartbeat/beater/heartbeat.go

```diff
@@ -184,7 +183,7 @@ func (bt *Heartbeat) Run(b *beat.Beat) error {
 	}
 
 	if bt.config.ConfigMonitors.Enabled() {
-		bt.monitorReloader = cfgfile.NewReloader(b.Info.Logger.Named("module.reload"), b.Publisher, bt.config.ConfigMonitors)
+		bt.monitorReloader = cfgfile.NewReloader(b.Info.Logger.Named("module.reload"), b.Publisher, bt.config.ConfigMonitors, b.Paths)
```

Comment:
If you remove this one file from this PR you get rid of the code owner request from the hosted services team ;)

Reply:
Yes.  But then we need a special case of `NewReloader` that doesn't take the extra argument, because `crawler.go` has references to `NewReloader` that need the new argument.

We can switch `b.Paths` to `paths.Paths` (ie. the global variable) if you want, but I'd much rather that all instances of `NewReloader` are "fixed".